td/tasks.py

```diff
@@ -1,32 +1,68 @@
 from __future__ import absolute_import
 
-from django.db import connection
-
 from celery import task
+from django.db import connection
```

Comment:
Not a big deal, but let's keep the style of this imports matching the [Pinax contribution guidelines](http://pinaxproject.com/pinax/how_to_contribute/#coding-style) where Django imports are in their own section after stdlib and before other external libraries.